classes/class-uagb-helper.php

```diff
@@ -1316,6 +1316,30 @@ public static function get_browser_name( $user_agent ) {
 				return 'ie';
 			}
 		}
+
+		/**
+		 * Get block dynamic CSS selector with filters applied.
+		 *
+		 * @param string $block_name Block name to filter.
+		 * @param array  $selectors Array of selectors to filter.
+		 * @param array  $attr Attributes.
+		 * @return array Combined selectors array.
+		 * @since X.X.X
+		 */
+		public static function get_combined_selectors( $block_name, $selectors, $attr ) {
+			if ( ! is_array( $selectors ) ) { 
+				return $selectors;
+			}
+
+			$combined_selectors = array();
+
+			foreach ( $selectors as $key => $selector ) {
+				$hook_prefix                = ( 'desktop' === $key ) ? '' : $key;
```

Comment:
```suggestion
				$hook_prefix                = ( 'desktop' === $key ) ? '' : '_'.$key;
```
please check if this will be required

Reply:
added commit